Formula/jfrog-cli-go.rb

```diff
@@ -31,8 +31,8 @@ def install
   end
 
   test do
-    actual = pipe_output("#{bin}/jfrog rt -v")
-    expected = "jfrog version 1.0.1\n"
+    actual = shell_output("#{bin}/jfrog -v").chomp
```

Comment:
`assert_equal "jfrog version 1.1.0", shell_output("#{bin}/jfrog -v").chomp` makes this a one-liner so would be preferable